contents/docs/surveys/_snippets/ios-surveys-installation.mdx

```diff
@@ -0,0 +1,62 @@
+## Step one: Add PostHog to your app
+
+> 🚧 **Note:** Mobile surveys for iOS is currently in **beta**. We'd love to hear your feedback on betas via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Asurveys%3Alow%3Atrue) or one of our other [support options](/docs/support-options).
+>
+> Using it requires PostHog's iOS SDK version >= 3.22.0, but it's recommended to always use the latest version.
+
+import IOSInstall from "../../integrate/_snippets/install-ios.mdx"
+
+<IOSInstall />
+
+
+## Step two: Enable surveys in your configuration
+
+To enable surveys in your iOS app, first, import PostHog with access to its experimental APIs:
+
+```swift
+@_spi(Experimental) import PostHog // Import with Experimental SPI
+```
+
+Second, enable surveys in your PostHog configuration:
+
+```swift
+let POSTHOG_API_KEY = "<ph_project_api_key>"
+// usually 'https://us.i.posthog.com' or 'https://eu.i.posthog.com'
+let POSTHOG_HOST = "<ph_client_api_host>"
+let config = PostHogConfig(apiKey: POSTHOG_API_KEY, host: POSTHOG_HOST) // TIP: host is optional if you use https://us.i.posthog.com
+
+// Surveys require iOS 15.0 or later
+if #available(iOS 15.0, *) {
+    config.surveys = true // TIP: If you see a "'surveys' is inaccessible due to '@_spi' protection level" build error, please see step 1 above.
+}
+
+PostHogSDK.shared.setup(config)
+```
+
+## Supported Features
+
+| Feature                           | Support                            |
+| --------------------------------- | ---------------------------------- |
+| **Questions**                     |                                    |
+| All question types                | ✅                                 |
+| Multi-question surveys            | ✅                                 |
+| Confirmation message              | ✅                                 |
+| Feedback button presentation      | ❌                                 |
+| **Customization / Appearance**    |                                    |
+| Set colors in PostHog dashboard   | ✅                                 |
+| Shuffle questions                 | ❌                                 |
+| PostHog branding                  | ❌                                 |
+| Delay popup after screen view     | ❌                                 |
+| Position config                   | ❌ Always bottom sheet             |
+| **Display conditions**            |                                    |
+| Feature flag & property targeting | ✅                                 |
+| Screen targeting                  | ❌                                 |
+| Device type targeting             | ✅                                 |
+| Survey wait period                | ❌                                 |
+| Event triggers                    | ✅                                 |
+| Action triggers                   | ❌                                 |
```

Comment:
can we match the RN table as this one? removing the CSS and adding the Screen targeting